ts/theme/components/View.ts

```diff
@@ -60,6 +68,8 @@ export default (): Theme => {
       paddingLeft: variables.footerPaddingLeft,
       paddingRight: variables.footerPaddingRight,
       paddingTop: variables.footerPaddingTop,
+      borderBottomWidth: variables.footerBottomBorderWidth,
```

Comment:
`footerBottomBorderWidth` -> `footerBorderBottomWidth` (for consistency with `borderBottomWidth`)

Reply:
@cloudify i'm working on zIndex usage. Actually, it seems not working, I think due to the two components do being siblings.

Reply:
weird, it seems like [zIndex only works within sibling components](https://facebook.github.io/react-native/docs/0.55/layout-props#zindex), see also [this thread](https://github.com/facebook/react-native/issues/18344), [this thread](https://github.com/facebook/react-native/issues/698)

Reply:
also [here](https://github.com/facebook/react-native/issues/8970)

Reply:
another option would be to kind of enable/disable the gradient based on some property of the route, but that would decouple the flag from the implementation of the screen so it may be confusing